stream-chat-android-core/src/main/java/io/getstream/chat/core/internal/fsm/builder/FSMBuilderMarker.kt

```diff
@@ -1,7 +0,0 @@
-package io.getstream.chat.core.internal.fsm.builder
-
-import io.getstream.chat.core.internal.InternalStreamChatApi
-
-@InternalStreamChatApi
-@DslMarker
-public annotation class FSMBuilderMarker
```

Comment:
This class was moved and not deleted, weird that the diff didn't pick this up correctly